app/services/invite_schools.rb

```diff
@@ -221,6 +221,15 @@ def send_induction_coordinator_add_participants_email
     end
   end
 
+  def send_year2020_invite_email
+    return unless FeatureFlag.active?(:year_2020_data_entry)
+
+    School.currently_open.each do |school|
```

Comment:
I read your comment, but looking at slack I think this should be `eligible`

Reply:
Yep, you got to it literal seconds before I fixed it :D